keentools_facebuilder/blender_independent_packages/pykeentools_loader/install.py

```diff
@@ -266,3 +327,33 @@ def module():
 
     import pykeentools
     return pykeentools
+
+
+caches_dir_path = module().utils.caches_dir()
+
+
+def download_file_path(part_installation):
+    file_name = download_file_name(part_installation)
+    return os.path.join(caches_dir_path, file_name)
+
+
+def updates_downloaded():
+    return os.path.exists(download_file_path(PartInstallation.CORE)) and \
+           os.path.exists(download_file_path(PartInstallation.ADDON))
+
+
+def remove_download(part_installation):
+    os.remove(download_file_path(part_installation))
+
+
+def install_download(part_installation):
```

Comment:
Are there any use cases when `remove_download`, `install_download` e.t.c are used for one part only? Why don't you join them? Also I don't like the names here